Unit.cpp

```diff
@@ -24,3 +24,27 @@ void Unit::takeDamage(const Unit& enemy)
 		hp = 0;
 	}
 }
+
+Unit* Unit::parseUnit(const std::string& filename){

```

Comment:
Nem rossz, csak gondoljatok at, hogy akarjatok-e igy tartani a dolgot. Gondolok itt arra, hogy most belul dinamikusan lefoglaltok egy fuggvenyt, amit remelitek, hogy majd valaki valamikor fel fog szabaditani. Most meg atlatjatok, hogy oke a main vegen felszabaditjatok. (Bar megjegyzem, ha az elso unit sikeresen letrejon, es a masodik hasal el exceptionnel, akkor azt nem.) Szoval ha dinamikusan dolgoztok, akkor ezekre a cuccokra majd nagyon kell figyelni.

Reply:
Tartjuk magunkat ehhez, igyekszünk figyelni ezekre a jövőbe, a Main-be külön try catch blokkba van a fájlok beolvasása, így amennyiben a másodiknál szállna el a program, az elsőt törli a catch-ben